e2e/scaffolding/helpers.ts

```diff
@@ -52,6 +52,11 @@ export const CENTS = 1000000n;
 export const DOLLARS = 100n * CENTS;
 export const STARTING_BALANCE = 6n * CENTS + DOLLARS;
 
+export function getTokenPerCapacity(): bigint {
+  // Perbil
+  return 1_000_000_000n / ExtrinsicHelper.api.consts.capacity.capacityPerToken.toBigInt();
```

Comment:
Is this token per capacity, or Planck per capacity? Or am I reading this wrong?

Reply:
You have a billion tokens

Reply:
It is capacity per token (from the chain) in parts per billion (thus the 1 billion) and it is in planks.